README.md

```diff
@@ -61,6 +61,27 @@ export GH_CLIENT_SECRET=<client secret>
 make cluster/prepare/local
 ```
 
+* 3Scale requires SMTP credentials to be able to send mail to users.
+
+**Note:** If this secret isn't created, the integreatly preflight checks will fail
```

Comment:
To confirm, this will only happen in `managed` installations?

Reply:
@aidenkeating yes, that's a good point. The preflight checks will skip the secret check for the other kind(s) of installation. That reminds me that I haven't tried with a non-`managed` installation.

Also, maybe I should add a dummy secret for the `make code/run`, since that's a managed installation. WDYT?

Reply:
@grdryn I think a dummy secret for `make code/run` makes sense as it'll make development simpler. I think people will likely be confused what secret this is if it blocks them all of a sudden.

Reply:
Dummy secret added.